src/GroupManager.php

```diff
@@ -93,6 +101,10 @@ public function addGroup($entity_type_id, $bundle_id) {
     $editable->set('groups', $groups);
     $saved = $editable->save();
 
+    // Notify other module we added a new group.
+    // todo: should this be an event?
+    $this->moduleHandler->invokeAll('og_group_created', [$entity_type_id, $bundle_id]);
```

Comment:
That's also a possibility! Very little overhead on our side, and it gives ultimate flexibility. I'm not aware of any places where this is done in core though.